services/core-web/src/components/Forms/AddPermitForm.js

```diff
@@ -1,9 +1,10 @@
+/* eslint-disable */
```

Comment:
Can we remove this and address any linting errors?

Reply:
yes, yes we can.

Reply:
@exzizt Fixed linting!